<?xml version="1.0"?>
<launch>
  <node name="servo_node" pkg="cable_reel" type="servo_node" output="screen">
    <param name="torque_enable" value="false"/>
  </node>

  <node name="poler_node" pkg="cable_reel" type="poler_control" output="screen"/>

  <!--
  <node name="joy_node" pkg="joy" type="joy_node" output="screen">
    <param name="dev" value="/dev/input/js0"/>
    <param name="deadzone" value="0.12"/>
    <param name="autorepeat_rate" value="20"/>
  </node>
  -->

</launch>